bottles/frontend/utils/connection.py

```diff
@@ -48,15 +49,23 @@ def check_connection(self, show_notification=False):
             return False
 
         try:
-            urllib.request.urlopen('https://repo.usebottles.com/components/index.yml', timeout=5)
+            res = subprocess.run(
+                ['curl', '-Is', 'https://repo.usebottles.com'],
```

Comment:
Is curl assumed to be available on every system?

Reply:
It's available in the flatpak which is enough